cv32e40x/regress/cv32e40x_compliance.yaml

```diff
@@ -8,10 +8,12 @@ description: Runs all RISCV compliance tests on the CV32E40X
 builds:
   uvmt_cv32e40x:
     cmd: make comp
+    cfg: no_bitmanip
     dir: cv32e40x/sim/uvmt
 
   uvmt_cv32e40x_compliance_build:
     cmd: make all_compliance
+    cfg: no_bitmanip
```

Comment:
AFAIK each compliance test targets specific ISA modules, and since they are written in assembly I would expect them (i.e. "imc" test etc) to not even use anything from B even if the compiler flag for B is on. But I don't know this for sure.

Reply:
Yes, I would _think_ so, but I also think this should eventually be updated to include the bitmanip extensions.